core/contracts/financial-templates/implementation/PricelessPositionManager.sol

```diff
@@ -366,6 +379,31 @@ contract PricelessPositionManager is FeePayer {
         emit SettleExpiredPosition(msg.sender, totalRedeemableCollateral.rawValue, tokensToRedeem.rawValue);
     }
 
+    /**
+    * @notice Premature contract settlement under emergency circumstances.
+    * @dev Only the governor can call this function as they are permissioned within the `FinancialContractAdmin`.
+    * Upon emergency shutdown, the contract settlement time is set to the shutdown time. This enables withdrawal
+    * to occur via the standard settleExpired function call. Contract state is set to `ExpiredPriceRequested`
+    * which prevents re-entry into this function or the `expire` function.
+    */
+    function emergencyShutdown() external onlyPreExpiration() onlyOpenState() {
```

Comment:
Think its wise to explicitly make a comment that we don't pay final fees here, for reason you said below